src/helpers/createApiDataBinding.ts

```diff
@@ -36,8 +37,8 @@ export const createApiDataBinding = (
     endpointKey: string, 
     bindingParams: EndpointParams = {}, 
     dispatch: ThunkDispatch<{ apiData: ApiDataState; }, void, Action>,
+    instanceId: string = '',
     bindingsStore: BindingsStore,
-    instanceId: string = ''
```

Comment:
is hier een reden voor (param volgorde veranderd)? instanceId is optional, dus die als laatst was logisch toch?